<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChatForm</class>
 <widget class="QWidget" name="ChatForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1190</width>
    <height>675</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QListView" name="lstContacts">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>10</y>
     <width>341</width>
     <height>661</height>
    </rect>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="txtChatList">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>821</width>
     <height>571</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtNewMessage">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>630</y>
     <width>761</width>
     <height>31</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Type your message here...</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnSend">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>630</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Send</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtCurrentContact">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>821</width>
     <height>31</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="placeholderText">
    <string>You can type the username that you want to chat with here...</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
